samples/subsys/tracing/prj_user.conf

```diff
@@ -1,2 +1,3 @@
 CONFIG_TRACING=y
 CONFIG_TRACING_USER=y
+CONFIG_IDLE_STACK_SIZE=2048
```

Comment:
Why change the stack size here, this seems unrelated to the rest

Reply:
The idle task stack is overflow on qemu_x86, qemu_cortex_m0 & qemu_riscv64. This makes the test fail. I added the description on the commit log. If this should separate to the other commit, please let me know.